text/0021-internal-data-monitoring.md

```diff
@@ -0,0 +1,182 @@
+---
+rfc pr: https://github.com/UCLA-Creative-Labs/cl-rfcs/pull/22
+tracking issue: https://github.com/UCLA-Creative-Labs/cl-rfcs/issues/21
+---
+
+# Watchdog
+
+`Watchdog` is a monorepo that will contain a variety of Creative Labs
+data analysis tools for internal use. 
+
+## Blurb
+
+The goal of this project is to allow all the teams across Creative Labs
+board to make data-driven decisions. We want to set up a system where all
+members can access previous data and use insights from the past to make
+good judgement calls in the present.
+
+Watchdog is meant to be a monorepo to house various data analysis tools
+for Creative Labs. There are a few ideas to implement currently. These
+projects are primarily meant for Creative Labs board members, but we
+could look towards displaying some of the data publicly or publishing
+our findings through a blog post for marketing content!
+
+Note: Since we do not have complete datasets on many events or projects in
+the past, much of this will be just to setup infrastructure for the future.
+
+### Project 1: Project Data Vis
+The first project is a continuation of the current Watchdog project, which
+is a tool to get data analytics on Creative Labs projects. Currently, it
+fetches project application information and displays its various statistics.
+We want to organize this by creating user datasets that would keep track
+of the same applicant through multiple application cycles. There would be a
+web app that displays the most important datasets in a visualized format and
+key conclusions would be highlighted. An addition would be to allow board
+members to better access the data by allowing users to choose their own
+method of displaying data (bar graph, pie chart, etc) and choosing a
+specific time period with various filters.
+
+### Project 2: Social Media Data Vis
+The second project would be a program that monitors how others interact
+with Creative Labs. This would be primarily to understand the correlation
+between social media posts interactions (ads, likes, shares) and event
+attendance. 
+
+### Project 3: Website Data Vis
```

Comment:
While this project is relatively simple, I would still like you to write out how it would function.

Questions to ask:
* Where to we put all the info we get?
* What is the data structure we are storing?
* How are we going to display this info? Or is that not part of this project?

Reply:
i dont believe this is resolved